core/controller/src/main/scala/whisk/core/controller/WebActions.scala

```diff
@@ -406,6 +413,10 @@ trait WhiskWebActionsApi
                         requestMethodParamsAndPath { context =>
                             provide(fullyQualifiedActionName(actionName)) { fullActionName =>
                                 onComplete(verifyWebAction(fullActionName, onBehalfOf.isDefined)) {
+                                    case Success((actionOwnerIdentity, action)) if context.method.value.toLowerCase == "options" && !action.annotations.asBool("web-custom-options").exists(identity) =>
```

Comment:
i think the `if` predicate here is too long and less readable - why not nest it below? (if not options ... else ...)